ui/src/components/AppDetailedToolbar.js

```diff
@@ -12,24 +12,22 @@ export class AppDetailedToolbar extends React.Component {
     this.props.history.push('/');
   }
 
-  onSaveApp = () => {
-    console.log('Save button clicked');
-  }
-
   render () {
+    const { app, onSaveApp, onDisableApp } = this.props;
```

Comment:
I see these are coming in from the parent component - can be add PropTypes in this class to add some checks for them? ie

```
AppDetailedView.propTypes = {
  app: PropTypes.object.isRequired,
  onSaveApp: PropTypes.func.isRequired,
  onDisableApp: PropTypes.func.isRequired
};
```

Reply:
Yep makes sense.